src/xdgmenumaker

```diff
@@ -291,9 +291,11 @@ def icon_full_path(icon):
     # in non-standard directories
     if os.path.exists(icon):
         if max_icon_size:
+	    if os.path.splitext(icon)[1] == ".svg":
```

Comment:
I'm sorry, I have some problems with spaces and tabs for indentation. Thank you for your answer.